modin/experimental/core/execution/native/implementations/hdk_on_native/dataframe/dataframe.py

```diff
@@ -238,7 +233,7 @@ def __init__(
         else:
             self.set_dtypes_cache(dtypes)
 
-        if partitions is not None:
+        if partitions is not None and partitions.size > 1:
```

Comment:
It doesn't make sense to call `_filter_empties()` in case of a single partition. Moreover, we should never have more than one partition in the HDK backend, that is not true at the moment and will be fixed in a separate PR. This code will be remover either.

Reply:
ok